ui/src/app/model/DbConfig.ts

```diff
@@ -0,0 +1,8 @@
+export default interface IDbConfig {
```

Comment:
file naming - specifically casing seems inconsistent

Reply:
renamed all the files to dash-case to match with other file names .

Reply:
There are few other files in the same folder like Conv.ts, Summary.ts, etc can you please rename them to start with lower case?

Reply:
These files have been renamed. Please review